### PR TITLE
Allow additional parameters to be passed to bigrquery

### PR DESCRIPTION
**Pull request by deflaux**
also
* add documentation
* check for an additional column name for alternate_bases

### CODE REVIEW

**Comment by deflaux** [comment]
@bthiruv please take a look at the documentation and let me know if you have any feedback, thanks!

**Comment by mbookman** [comment]
LGTM

File: R/PlatinumGenomes-QC/rHelpers/setup.R [hunk 1]
```diff
@@ -19,7 +19,32 @@ require(scales)
...
+#'
+#' @examples
+#' DisplayAndDispatchQuery('https://raw.githubusercontent.com/googlegenomics/codelabs/master/R/PlatinumGenomes-QC/sql/private-variants.sql',
+#'                         project=YOUR-PROJECT_ID,
+#'                         replacements=list('_GENOME_CALL_TABLE_'='genomics-public-data:platinum_genomes.variants'))
+#'
+#' DisplayAndDispatchQuery('https://raw.githubusercontent.com/googlegenomics/codelabs/master/R/PlatinumGenomes-QC/sql/private-variants.sql',
+#'                         project=YOUR-PROJECT_ID,
+#'                         replacements=list('_GENOME_CALL_TABLE_'='genomics-public-data:platinum_genomes.variants'),
+#'                         destination_table='deflaux'YOUR-DATASET.YOUR-NEW-TABLE')
```
Comment (pgrosu) [inline R/PlatinumGenomes-QC/rHelpers/setup.R]: s/'deflaux'YOUR-DATASET.YOUR-NEW-TABLE'/'YOUR-DATASET.YOUR-NEW-TABLE'

File: R/PlatinumGenomes-QC/rHelpers/setup.R [hunk 2]
```diff
@@ -44,9 +69,19 @@ DisplayAndDispatchQuery <- function(queryUri, project, replacements=list()) {
   cat(querySql)
 
   # Dispatch the query to BigQuery.
-  query_exec(querySql, project)
+  query_exec(querySql, project, ...)
 }
 
+#' This is a helper method to display the first few rows of query results.
```
Comment (pgrosu) [inline R/PlatinumGenomes-QC/rHelpers/setup.R]: s/of query/of the query

File: R/PlatinumGenomes-QC/rHelpers/setup.R [hunk 3]
```diff
@@ -54,7 +89,8 @@ DisplayQueryResults <- function(result, n=6) {
     # Suppress the printing of sample identifiers when the data contains allele
     # values.  This is brittle since the query could use a different name for the columns
     # containing the identifier or allele.
-    if(2 == length(intersect(c("call_call_set_name", "alternate_bases"), colnames(result)))) {
+    if(2 == length(intersect(c("call_call_set_name", "alternate_bases", "alt.alternate_bases"),
+                             colnames(result)))) {
       result <- mutate(result, call_call_set_name = "not displayed")
```
Comment (pgrosu) [inline R/PlatinumGenomes-QC/rHelpers/setup.R]: Why not just have the following, in order to exclude the column automatically:

```R
result <- subset(result, select=-c(call_call_set_name))
```
It basically performs the following:
```R
> result
  call_call_set_name alternate_bases alt.alternate_bases
1                  1               8                  15
2                  2               9                  16
3                  3              10                  17
4                  4              11                  18
> result <- subset(result, select=-c(call_call_set_name))
> result
  alternate_bases alt.alternate_bases
1               8                  15
2               9                  16
3              10                  17
4              11                  18
> 
```

**Comment by bthiruv** [comment]
Thanks, Nicole.

**Pull request merged**